README.md

```diff
@@ -45,41 +56,187 @@ cat some-css-file.css | cssstats
+The following options add the results of helper methods to the returned object. This is helpful when using `JSON.stringify()`.
 
-__`rules`:__ Flattened array of rules from PostCSS.
+- `specificityGraph` (boolean, deault `false`)
+- `sortedSpecificityGraph` (boolean, deault `false`)
+- `repeatedSelectors` (boolean, deault `false`)
+- `propertyResets` (boolean, deault `false`)
+- `vendorPrefixedProperties` (boolean, deault `false`)
 
-__`aggregates`:__ Aggregate data for the entire stylesheet.
-- `selectors` - total number of selectors
-- `declarations` - total number of declarations
-- `properties` - an array of properties used in the stylesheet
-- `mediaQueries` - an array of media query strings used in the stylesheet
-- `idSelectors` - total number of selectors containing an id
-- `classSelectors` - total number of selectors containing a class
-- `pseudoElementSelectors` - total number of selectors containing an pseudo element
-- `pseudoClassSelectors` - total number of selectors containing a pseudo class
-- `repeatedSelectors` - array of selectors that were declared more than once
+### Returned Object
 
-For every unique property found in the stylesheet, `aggregates` also includes these values:
-- `[property].total` - total number of [property] declarations
-- `[property].unique` - number of unique [property] declarations
+```js
+// Example
+{
+  size: n,
+  gzipSize: n,
+  rules: {
+    total: n,
+    size: {
+      graph: [n],
+      max: n,
+      average: n
+    }
+  },
+  selectors: {
+    total: n,
+    id: n,
+    class: n,
+    type: n,
+    pseudoClass: n,
+    psuedoElement: n,
+    values: [str],
+    specificity: {
+      max: n
+      average: n
+    },
+    getSpecificityGraph(),
+    getRepeatedValues(),
+    getSortedSpecificity()
+  },
+  declarations: {
+    total: n,
+    important: n,
+    properties:
+      prop: [str]
+    },
+    getPropertyResets(),
+    getUniquePropertyCount(),
+    getPropertyValueCount(),
+    getVendorPrefixed(),
+    getAllFontSizes(),
+    getAllFontFamilies(),
+  },
+  mediaQueries: {
+    total: n,
+    unique: n,
+    values: [str],
+    contents: [
+      {
+        value: str,
+        rules: {
+          total: n,
+          size: {
+            graph: [n],
+            max: n,
+            average: n
+          }
+        },
+        selectors: {
+          total: n,
+          id: n,
+          class: n,
+          type: n,
+          pseudoClass: n,
+          pseudoElement: n,
+          values: [str],
+          specificity: {
+            max: n,
+            average: n
+          }
+        },
+        declarations: {
+          total: n,
+          important: n,
+          vendorPrefix: n,
+          properties: {
+            prop: [str]
+          }
+        }
+      }
+    ]
+  }
+}
+```
+
+#### `size`
+The size of the file in bytes
+
+#### `gzipSize`
+The size of the stylesheet gzipped in bytes
+
+#### `rules` object
+
+- `total` - total number of rules
+- `size` object
```

Comment:
I like the current formatting. Though, I'd probably prefer to see the type for all properties: `number`, `object`, `array`, etc. IMO it's pretty useful when scanning the API docs.